src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac img"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__info">Product code: 195434</p>
+
+      <div class="card__rate rate">
+        <div class="rate__stars">
+          <div class="rate__stars-star"></div>
+          <div class="rate__stars-star"></div>
+          <div class="rate__stars-star"></div>
+          <div class="rate__stars-star"></div>
+          <div class="rate__stars-star"></div>
```

Comment:
Fix classes and add modifier classes 
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

Reply:
![image](https://github.com/user-attachments/assets/986b53d8-03ae-48ee-ba99-ed1ee8f03623)
I don't understand why I need a modifier if I don't use it.
I could but it would increase the lines in my code